pyproject.toml

```
@@ -35,6 +35,7 @@ dev = [
     "pytest",                       # Testing framework
     "flake8",                       # Code linter
     "pytest-cov",                   # Test coverage plugin for pytest
+    "pydocstyle"                    # Docstrings linter
```

```suggestion
    "pydocstyle",                    # Docstrings linter
```
Add commas for better git diffing